### PR TITLE
Ncgames 5

### PR DESCRIPTION
**Pull request by Greeny1248**
Task 5 completed :review_id

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -4,17 +4,26 @@ const app = express();
...
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+});
+app.use((req, res, next) => {
+  res.status(404).send({ msg: "Route Does Not Exist" });
   next(err);
 });
```
Comment (NRMSMITH) [inline app.js:46]: I think you could consider splitting off these into a different file to keep it all neat at this point. An error handling file is always a good way to organise it all off :)

File: controller/controller.js [hunk 2]
```diff
@@ -21,4 +25,14 @@ const viewAllReviews = (req, res) => {
     });
 };
 
-module.exports = { viewAllCategories, viewAllReviews };
+const viewReviewById = (req, res, next) => {
+  const { review_id } = req.params;
+  console.log(req.params, "Req");
```
Comment (NRMSMITH) [inline controller/controller.js]: Avoid leaving console logs in please!

File: __tests__/app.test.js [hunk 3]
```diff
@@ -66,19 +66,36 @@ describe("app", () => {
...
+              expect(review.hasOwnProperty("created_at")).toBe(true);
+              expect(review.hasOwnProperty("votes")).toBe(true);
+              expect(review.hasOwnProperty("designer")).toBe(true);
+              expect(review.hasOwnProperty("review_body")).toBe(true);
+            });
+        });
+      });
     });
   });
 });
```
Comment (NRMSMITH) [inline __tests__/app.test.js:102]: Where are your error handling tests? You should do one for a 400 and one for a 404.

**Review by NRMSMITH — CHANGES_REQUESTED** [review]
You need to do some testing for your errors.

File: controller/controller.js (hunk 2 above)
Comment (Greeny1248) [reply controller/controller.js]: deleted!

**Pull request merged**